### DISCUSSION TITLE
openTX futaba FASST C1-C2

### DISCUSSION THREAD

**Discussion by caio59** [pascallanger/DIY-Multiprotocol-TX-Module#605] [General]
Hi, 
is there in Jumper T16 opentx software any protocol compatible with FASST C1/C2 FUTABA ??

I've two receivers with that protocol:
 FRSKY TFR8SB
 GF400F Hobby King 

I don't be able to bind with any protocol.
I have the last firmware v1.3.2.86
thanks Claudio

**Comment by pascallanger** [comment]
FAAST is not supported...

**Reply by caio59** [reply]
thanks
 have I to buy an external module ?
could give me an advice ?

**Reply by kilrah** [reply]
There is no module that supports FASST nowadays, those that existed were discontinued more than 10 years ago.
